ToDoApp/Backend/src/resolvers/todo/todoResolver.js

```diff
@@ -1,68 +1,95 @@
-const { find }= require('lodash');
-const { generateIntID }= require("../../../utils.js");
+const { generateUUID }= require("../../../utils.js");
 const jwt = require('jsonwebtoken')
 const { CONFIG }= require("../../config/config");
+const { neo4jgraphql } = require('neo4j-graphql-js');
 
-let todos = [{
-        id: 1,
-        message: 'first todo',
-        finished: false,
-        creator: 1
-    },
-    {
-        id: 2,
-        message: 'second todo',
-        finished: true,
-        creator: 2
-    },
-];
-
-
-// Resolvers define the technique for fetching the types defined in the
-// schema. This resolver retrieves books from the "books" array above.
 const todoResolver = {
     Query: {
-        allTodos: () => todos,
-        todoById: (root, args,) => {
-            return find(todos, { id: args.id });
+        allTodos: async (parent, args, context) => {
+         const session = context.driver.session();
+         const queryResults = await session.run('MATCH(t:Todo) RETURN t LIMIT 1');
+         const todos = queryResults.records.map(todo => todo.get('t').properties);
+         session.close()
+         return todos;
         },
+        todoById: async (root, {id}, context) => {
+                  const session = context.driver.session();
+                  const queryResults = await session.run('MATCH (t:Todo {id: $id}) RETURN t',
```

Comment:
If you write
```suggestion
                  const queryResults = await session.run('MATCH (t:Todo {id: $id}) RETURN t {.*}',
```
you can skip the call of `.properties` on line 22 :point_down: